Header header
string type
float64 x
float64 y
float64 z
float64 theta
bool vertical
string shape
bool left
bool forward
bool right
